codechecker_common/output/gerrit.py

```diff
@@ -142,17 +144,19 @@ def __get_changed_files(changed_file_path: Union[None, str]) -> List[str]:
     changed_files = []
 
     if not changed_file_path or not os.path.exists(changed_file_path):
+        LOG.warning("CC_CHANGED_FILES is specified but the file (%s) doesn't "
+                    "exist!", changed_file_path)
         return changed_files
 
-    with open(changed_file_path,
-              encoding='utf-8',
-              errors='ignore') as changed_file:
-        content = changed_file.read()
+    with open(changed_file_path, encoding='utf-8', errors='ignore') as f:
+        content = f.read()
 
         # The file can contain some garbage values at start, so we use
         # regex search to find a json object.
         match = re.search(r'\{[\s\S]*\}', content)
         if not match:
+            LOG.debug("The content of the given changed file (%s) is invalid! "
+                      "%s", changed_file_path, content)
```

Comment:
Maybe if the file content is wrong, it can produce large output in the log. So I suggest to remove "content" from trhe log message.